pty_windows.go

```diff
@@ -0,0 +1,160 @@
+package pty
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
```

Comment:
Probably yes, the system32.dll has a lot of syscall definitions, and if not found, usually the program should not be running at all.

Of course we can load these syscalls and return error when actually calling them, and for compatibility reason, this could be better.

Reply:
Now we will only load `kernel32.dll` in package init(), all other syscalls are loaded during function call.